realtime/server.js

```diff
@@ -77,8 +77,11 @@ socket.on('connection', function (client) {
             return;
         }
         people[client.id] = username;
+        var people_order = [];
+        for (var id in people)
```

Comment:
For loop can be written in one line using [_.values](https://lodash.com/docs#values)